js/grid/generate_grid.js

```diff
@@ -106,7 +111,12 @@ function appendTableData(trFall, trSpring, time) {
 
     var weekPrefixArray = ["M", "T", "W", "R", "F"];
 
-    var adjustedTime = (time === 12 ? 12 : time % 12) + ':00';
+    if (time % 1 == 0){
```

Comment:
Please put a space between the final `)` and the `{`.
Also, we've been using the [`===` over the `==`.] (http://stackoverflow.com/questions/523643/difference-between-and-in-javascript)